codegen/src/api/calls.rs

```diff
@@ -35,13 +34,51 @@ use quote::{
 };
 use scale_info::form::PortableForm;
 
+/// Generate calls from the provided pallet metadata.
```

Comment:
```suggestion
/// Generate calls from the provided pallet's metadata.
```

Reply:
I suggested this in a few places. It might be wrong, but I wasn't sure if this was referring to a single pallet, or the general pallet metadata as a hole. Feel free to ignore/resolve this if I'm wrong :)